nss-cohort-40/banagzon-workforce-mgt-lumberjacks
# Employee computer assignment

# Description
Added functionality for assigning an employee when purchasing a computer

 Fixes # [13](https://github.com/nss-cohort-40/banagzon-workforce-mgt-lumberjacks/issues/13)

 ## Type of change
 Please delete options that are not relevant.

 # Testing Instructions
Please try adding a new computer and assigning an employee. Make sure the employee's name is then displayed under the new computer in the computer list.

 # Checklist:
